client/coral-admin/src/graphql/fragments/commentView.graphql

```diff
@@ -12,4 +12,13 @@ fragment commentView on Comment {
       id
       title
     }
+    actions {
+      ... on FlagAction {
+        reason
+        message
```

Comment:
by extending this you are increasing the payload in the EmbedStream since it uses the same fragment